web/src/upload.ts

```diff
@@ -274,6 +274,20 @@ export function rewire_upload_files(value: typeof upload_files): void {
     upload_files = value;
 }
 
+export function upload_pasted_file(pasted_file: File): void {
+    if ($("textarea").hasClass("message_edit_content")) {
```

Comment:
This code is nonsense -- you need to pass in the actual textarea that the person is pasting into, not do something just select all textareas.

@kuv2707 would you be up for picking up this PR and submitting a cleaned up version of it?